packages/platform-android/native_modules.gradle

```diff
@@ -2,6 +2,7 @@ import groovy.json.JsonSlurper
 import org.gradle.initialization.DefaultSettings
 import org.apache.tools.ant.taskdefs.condition.Os
 
+def jsAppDir = buildscript.getProperties().get("sourceFile").toString().split("node_modules/@react-native-community/cli-platform-android/native_modules.gradle")[0]
```

Comment:
Thinking about it again, we don't need the app folder. Npx should be fine with running from sourceFile dir as well, which should simplify this a bit. Will update later

Reply:
Wrong trait, packages in node_modules also have `package.json`s, so it won't work with current project root discovery.